backend/app/api/endpoints/openapi_responses.py

```
@@ -294,95 +294,481 @@ async def create_response(
+            team=team,
+            user=user,
+            message=input_text,
+            enable_tools=enable_chat_bot,
+            enable_deep_thinking=enable_chat_bot,
+            enable_web_search=enable_chat_bot and settings.WEB_SEARCH_ENABLED,
         )
-    except HTTPException:
-        raise
     except Exception as e:
-        logger.error(f"Failed to create task: {str(e)}")
+        logger.error(f"Failed to build execution request: {e}")
```

Ruff (0.14.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)